pgrx-sql-entity-graph/src/pg_extern/mod.rs

```diff
@@ -387,20 +383,16 @@ impl PgExtern {
         let arg_fetches = args.iter().enumerate().map(|(idx, arg)| {
             let pat = &arg_pats[idx];
             let resolved_ty = &arg.used_ty.resolved_ty;
-            if arg.used_ty.resolved_ty.to_token_stream().to_string() == quote!(pgrx::pg_sys::FunctionCallInfo).to_token_stream().to_string()
-                || arg.used_ty.resolved_ty.to_token_stream().to_string() == quote!(pg_sys::FunctionCallInfo).to_token_stream().to_string()
-                || arg.used_ty.resolved_ty.to_token_stream().to_string() == quote!(::pgrx::pg_sys::FunctionCallInfo).to_token_stream().to_string()
-            {
-                quote_spanned! {pat.span()=>
+            match resolved_ty {
+                // There's no danger of misinterpreting the type, as pointer coercions must typecheck!
+                ty @ Type::Path(_) if last_ident_is(ty, "FunctionCallInfo") => quote_spanned! {pat.span()=>
                     let #pat = #fcinfo_ident;
-                }
-            } else if arg.used_ty.resolved_ty.to_token_stream().to_string() == quote!(()).to_token_stream().to_string() {
-                quote_spanned! {pat.span()=>
+                },
+                Type::Tuple(tup) if /* () */ tup.elems.is_empty() => quote_spanned! {pat.span()=>
```

Comment:
nit: delete the `/* () */`, and add spaces around `pat.span()`.

Reply:
```suggestion
                Type::Tuple(tup) if tup.elems.is_empty() => quote_spanned! { pat.span() =>
```